### PR TITLE
[4.0] Fix numerics

### PR DESCRIPTION
**Pull request by brianteeman**
This PR fixes the bug reported in #35539 **AND** removes the Filter Numeric Terms option and related code as it is not complete and doesnt do anything at all

### CODE REVIEW

File: administrator/components/com_finder/src/Indexer/Query.php [hunk 1]
```diff
@@ -1283,11 +1283,6 @@ protected function processString($input, $lang, $mode)
 					continue;
 				}
 
-				if ($params->get('filter_numerics', 0) && $token->numeric)
```
Comment (Quy) [inline administrator/components/com_finder/src/Indexer/Query.php:1286]: There are 5 instances without `s` in this file.:

https://github.com/joomla/joomla-cms/blob/32f6c81fc2f1d35bec3853f345a93421d93dba6f/administrator/components/com_finder/src/Indexer/Query.php#L1010

Comment (brianteeman) [reply administrator/components/com_finder/src/Indexer/Query.php:1286]: even more proof that this code never worked and was never tested by the original submitter and it should just be removed

**Comment by alikon** [comment]
i'm lost...
what's the relation between #35339 and this ?

**Comment by brianteeman** [comment]
oops wrong PR - updated now

File: administrator/components/com_finder/src/Indexer/Indexer.php [hunk 2]
```diff
@@ -956,7 +950,11 @@ protected function addTokensToDb($tokens, $context = '')
 				++$values;
 			}
 
-			$db->setQuery($query)->execute();
+			// Only execute the query if there are tokens to insert
```
Comment (PhilETaylor) [inline administrator/components/com_finder/src/Indexer/Indexer.php:953]: This change seems to be the same as #35540 ?

**Comment by richard67** [comment]
@brianteeman Now as PR #35540 has been merged, I would expect this PR here to show a conflict because it contains the same change as in PR #35540 . But it doesn't show any conflict on GitHub, and it still shows the change from PR #35540 , too. Could you update your branch to the 4.0-dev of the CMS to solve that? Or if you don't have the time, allow me to do that? I'd like that to be solved before I test.

**Comment by brianteeman** [comment]
.

**Comment by richard67** [comment]
@brianteeman Did you close it by purpose, and will you make a new one? Maybe I can try to fix our branch, but the easiest thing would be to make a new one, I think.

**Pull request closed without merging**